pkg/util/log/log.go

```diff
@@ -43,12 +43,21 @@ var (
 	defaultStackDepth    = 3
 )
 
+type LogLevelSource string
+
+const (
+	LogLevelSourceCLI     LogLevelSource = "cli"
+	LogLevelSourceRC      LogLevelSource = "rc"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** 
```suggestion
	LogLevelSourceRC      LogLevelSource = "remote-config"
```